internal/test/e2e/e2e_test.go

```diff
@@ -204,11 +197,8 @@ func installOperator(t *testing.T, kubectl *kubectlContext, kind *cluster.Contex
 	go func() {
 		defer wg.Done()
 		t.Log("Installing cert-manager")
-		err := kubectl.Apply("--validate=false", "--filename=https://github.com/jetstack/cert-manager/releases/download/v0.11.0/cert-manager.yaml")
-		require.NoError(t, err)
-		t.Log("Waiting for cert-manager to be ready")
-		err = kubectl.Wait("--for=condition=Available", "--timeout=300s", "apiservice", "v1beta1.webhook.cert-manager.io")
-		require.NoError(t, err)
+		out, err := exec.CommandContext(ctx, "make", "-C", *fRepoRoot, "deploy-cert-manager").CombinedOutput()
+		require.NoError(t, err, string(out))
```

Comment:
There is now a single way to install cert-manager in the Makefile, and I call that here instead.
Should allow us to parameterize the tests with different versions of cert-manager, in future.